itou/siaes/models.py

```diff
@@ -614,7 +614,17 @@ def display_location(self):
     def display_contract_type(self):
         return self.other_contract_type or self.get_contract_type_display
 
+    @property
+    def is_external(self):
+        return self.source_kind == JobSource.PE_API
```

Comment:
Est-ce qu'avec le changement sur le model, maintenant ça serais pas juste `return self.source_kind is not None` ?

Reply:
tutafé